src/utils/commits-utils.ts

```diff
@@ -131,3 +133,29 @@ export const showPLRMessage = (plr: PipelineRunKind): string => {
     return 'Releasing';
   }
 };
+
+export const createRepoUrl = (commit: Commit): string | null => {
+  if (commit.repoURL) {
+    return commit.repoURL;
+  }
+  if (commit.repoName && commit.repoOrg) {
+    return `https://github.com/${commit.repoOrg}/${commit.repoName}`;
+  }
+  return null;
+};
+
+export const createRepoBranchURL = (commit: Commit): string | null => {
+  const repoUrl = createRepoUrl(commit);
+  if (commit.branch && repoUrl) {
+    return `${repoUrl}/tree/${commit.branch}`;
```

Comment:
Do we need to limit this to github ?

Reply:
moved git provider check inside the util